css/webcodebook.css

```diff
@@ -4,11 +4,27 @@ General
 
 .web-codebook {
   position: relative;
+  width:1024px;
 }
 
 .web-codebook > * {
     display: block;
-    width:1024px;
+    width: 100%;
+}
+
```

Comment:
Let's add a 2nd 'background' div for the spinner with `background:white` and `opacity:0.X`